utils.go

```
@@ -2,13 +2,32 @@
 
 import "github.com/shogo82148/ints"
 
-// squash16 squashes the bits of x to a single bit.
-func squash16(x uint16) uint16 {
-	x |= x >> 8
-	x |= x >> 4
-	x |= x >> 2
-	x |= x >> 1
-	return x & 1
+func zero16(x uint16) uint16 {
```

golangci-lint (1.64.8)

func `zero16` is unused

(unused)